documentation/library_guide/api_for_dpcpp_kernels/tested_standard_cpp_api.rst

```diff
@@ -237,7 +237,7 @@ C++ Standard API                     libstdc++  libc++     MSVC
 ------------------------------------ ---------- ---------- ----------
 ``std::ratio``                       Tested     Tested     Tested
 ------------------------------------ ---------- ---------- ----------
-``std::complex``                     Tested                Tested
+``std::complex``                     Tested     Tested     Tested
```

Comment:
Are we sure that `complex` is tested and passed in full for libc++?

Reply:
From my point of view, testing scope of std::complex is the same for libstdc++,  libc++ and  MSVC.
But there are no tests for some operations like
https://en.cppreference.com/w/cpp/numeric/complex/operator_arith,
https://en.cppreference.com/w/cpp/numeric/complex/operator_arith2,
https://en.cppreference.com/w/cpp/numeric/complex/operator_arith3,
https://en.cppreference.com/w/cpp/numeric/complex/operator_cmp,
https://en.cppreference.com/w/cpp/numeric/complex/operator%3D,
https://en.cppreference.com/w/cpp/numeric/complex/operator_ltltgtgt
and some other functions.

Reply:
Done

Reply:
Done